frontend/src/charts/choroplethMap/mapLegendUtils.ts

```diff
@@ -117,11 +117,15 @@ export const createUnknownLegend = (
   })
 }
 
-export function useGetLegendColumnCount() {
+export function useGetLegendColumnCount(isMulti?: boolean) {
   const isCompareMode = window.location.href.includes('compare')
   const isTiny = useIsBreakpointAndUp('tiny')
   const isSm = useIsBreakpointAndUp('sm')
+  const isLg = useIsBreakpointAndUp('lg')
+  if (isLg && isMulti) return 10
```

Comment:
great! you can use your judgement, and if you want to adjust that down to `'md'` or potentially add some more options to this function to really make the best layout feel free